src/components/organisms/AssetsList.tsx

```diff
@@ -95,23 +101,39 @@ const AssetssList = ({ data }: Props) => {
       ),
     },
     {
-      title: "Change",
+      title: "Change today",
       dataIndex: "change_today",
-      key: "change_today",
       width: 200,
       render: (change_today: any) => (
         <PercentageDisplay
           indicator={
             change_today > 0
               ? INDICATOR.POSITIVE
               : change_today === 0
-                ? INDICATOR.NEUTRAL
-                : INDICATOR.NEGATIVE
+              ? INDICATOR.NEUTRAL
```

Comment:
Looks like your prettier or eslint settings are not aligned with the projects settings. These indetations should be kept.